.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
+jobs:
+  publish:
+    docker:
+        - image: python:3.6
+    steps:
+      - checkout
+      - run: python -m pip install --user --upgrade setuptools wheel
+      - run: python setup.py sdist bdist_wheel
```

Comment:
should we expect this to be handled when submitting changes for a PR as opposed to during release? I would add a step for the dev making the changes but would allow us to ensure that nothing we don't want gets into the historical versions